pulp_container/tests/functional/utils.py

```diff
@@ -73,12 +75,19 @@
 ]
 
 
-def gen_user(permissions):
+def gen_user(model_permissions=None, object_permissions=None):
```

Comment:
model_permissions = []
this will eliminate the need for the lines 80-85

Reply:
I am not going to use `list` as a default argument here. It could possibly create some issues in the future when calling the method more than once (https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments).